### PR TITLE
Remove 19h1 and add 21H1 to Helix test queue

### PR DESCRIPTION
**Pull request by kythant**

### CODE REVIEW

**Comment by jefgen** [comment]
Note: The PR into `main` is here: https://github.com/microsoft/WindowsAppSDK/pull/2038

Does the change into `release` need a servicing bug? Not sure for Foundation repo.

**Review by jefgen — APPROVED** [review]
LGTM. Not sure if a bug is needed though.

File: build/AzurePipelinesTemplates/WindowsAppSDK-RunHelixTests-Job.yml [hunk 1]
```diff
@@ -30,8 +30,8 @@ jobs:
 
     # The target queues to run the tests on.
     # Note: %3b is the escape sequence for ';' which is used as the delimiter
-    helixTargetQueuesOpen: 'Windows.10.Amd64.ClientRS5.Open.reunion%3bWindows.10.Amd64.Client19h1.Open.reunion%3bWindows.10.Amd64.Client20h2.Open.reunion'
-    helixTargetQueuesClosed: 'Windows.10.Amd64.ClientRS5.reunion%3bWindows.10.Amd64.Client19h1.reunion%3bWindows.10.Amd64.Client20h2.reunion'
+    helixTargetQueuesOpen: 'Windows.10.Amd64.ClientRS5.Open.reunion%3bWindows.10.Amd64.Client20h2.Open.reunion%3bwindows.10.amd64.client21h1.open.reunion'
+    helixTargetQueuesClosed: 'windows.10.amd64.client20h2.reunion%3bwindows.10.amd64.clientrs5.reunion%3bwindows.10.amd64.client21h1.reunion'
```
Comment (DrusTheAxe) [inline build/AzurePipelinesTemplates/WindowsAppSDK-RunHelixTests-Job.yml:34]: Why change Closed to all lowercase but Open is still mixed case?

Inconsistent by accident or design?

**Pull request merged**